remoteappmanager/static/js/home/views/application_view.js

```diff
@@ -1,170 +1,91 @@
-            html = self._render_form(index);
-        } else {
-            $("section.content").addClass("no-padding");
-            html = self._render_app(index, delayed);
-        }
-        return html;
-    };
-
-    ApplicationView.prototype._render_form = function(index) {
-        // Renders the configuration form
-        var self = this;
-        var app_data = self.model.app_list[index].app_data;
-        var app_status = self.model.app_list[index].status;
-
-        var configurables = self.model.app_list[index].configurables;
-
-        var disabled = (app_status === models.Status.STARTING);
 
-        var base = $(templates.app_start_panel({
-            app_data: app_data,
-            index: index,
-            disabled: disabled
-        }));
-
-        base.find(".start-button").click(function() {
-            self.start_button_clicked($(this).attr("data-index"));
-        });
-
-
-        var policy = app_data.image.policy;
-        var policy_ul = base.find(".policy");
-
-        if (policy.allow_home) {
-            policy_ul.append($("<li>Workspace accessible</li>"));
-        } else {
-            policy_ul.append($("<li>Workspace not accessible</li>"));
-        }
-
-        if (policy.volume_source && policy.volume_target && policy.volume_mode) {
-            policy_ul.append($(
-                "<li>Volume mounted: " + policy.volume_source +
-                " &#x2192; " + policy.volume_target +
-                " (" + policy.volume_mode + ")</li>"));
-        } else {
-            policy_ul.append($("<li>No volumes mounted</li>"));
-        }
-
-        var fieldset = base.find("fieldset");
-
-        if (configurables.length === 0) {
-            fieldset.html("No configurable options for this image");
-        } else {
-            configurables.forEach(function(configurable) {
-                var widget = configurable.view(index);
-                fieldset.append(widget);
-            });
-        }
-
-        return base;
-    };
-
-    ApplicationView.prototype._render_app = function(index, delayed) {
-        // Renders the iframe with the application.
-        var self = this;
-        var app_data = self.model.app_list[index].app_data;
-        var location = urlutils.path_join(self.base_url,
-            "containers",
-            app_data.container.url_id
-        );
-
-        if (!delayed) {
-            location = location+"/";
-        }
-
-        var iframe_size = utils.max_iframe_size();
-        var iframe = $('<iframe id="application" frameBorder="0" ' +
-            'src="' + location + '" ' +
-            'style="min-width: '+iframe_size[0]+'px; min-height: '+iframe_size[1]+'px;"></iframe>');
-        return iframe;
-    };
+    var ga = gamodule.init();
+    var Status = utils.Status;
+
+    var ApplicationView = Vue.extend({
+        el: 'div.content-wrapper',
+
+        data: function() {
+            return {
+                model: { app_list: [], selected_index: null }
```

Comment:
probably, like the other one, is irrelevant, because you pass the model from outside.